c_tests/tests/compile_globals.c

```diff
@@ -1,5 +1,6 @@
 // Compiler:
 // Run-time:
+// ignore: Mutable global variables not supported.
```

Comment:
In general I reckon `ignore` tags should go at the top of the file so that people can't miss them.

Reply:
Fixed in 274a5b9.